virtual-cluster/alex.md

```diff
@@ -22,9 +23,33 @@
 > - Update repository and system: `apt update && apt dist-upgrade`
 > - Now that all the preparation has been done, proceed to install Proxmox: `apt-get install proxmox-ve`
 > - Go to GCP --> VPC network --> firewall rules and create a new firewall rule for tcp:8006. This is to allow access to Proxmox VE web interface.
-> - Done! Access the Proxmox admin web interface by typing https://youripaddress:8006 in the web browser.  
+> - Done! Access the Proxmox admin web interface by typing https://google-external-IP-address:8006 in the web browser.  
 >
 >Upon reaching this point, a Debian nested VM hosting Proxmox VE would be created.
 ***
 ### *Creating a cluster and adding all nodes*
-> Created 2 more VM instances based on the previous step to serve as nodes for the VM cluster. Tried to set up a cluster using the Proxmox web interface but was faced with a lot of issues during the process of adding the 2nd node to the cluster.
+> Created 2 additional VM instances based on the previous steps to set up for a 3 nodes VM cluster. Select a node to create the cluster and subsequently adding the rest of the nodes into the cluster. Below documents the steps to creating the "cloudcluster" proxmox cluster:  
+> - Node 1 is selected to create the cluster. Create the cluster by typing `pvecm create cloudcluster` in Node 1 CLI.
+> - Edit the etc/pve/corosync file based on the instructions stated in the **Solutions** segment below.
+> - Add Node 2 & Node 3 into the cluster by typing `pvecm add node-1` in Node 2 & 3 CLI.  
+> - The cluster "cloudcluster" will be successfully created with all 3 nodes connected.
+>
+>Even though the setting up of a cluster is simple and only requires a few command, it is only made possible because of the time and effort spend on researching and troubleshooting the problems faced. During the process of setting up a cluster consisting 3 nodes using the Proxmox web interface, a lot of issues arises during the process of adding the nodes into the cluster.  
+>
+>**Issues found:**  
+> 1) Netmask of VMs created on GCP are 255.255.255.255, which interferes with the setting up of promox clustering.
+>  
+> 2) GCP does not support multicast.
+>
+>**Solutions:**  
+>
+> 1) Due to the default netmask being 255.255.255.255, the corosync file is only able to work on the node which created the file. When the file is sync with another node, error will occur on the corosync service thus preventing the creation of the cluster.
+> - To rectify the netmask issue, add `--guest-os-features MULTI_IP_SUBNET` in to the gcloud command for creating nested VM image. (This command has been updated on to the gcloud command for creating the custom nested VM image in the ***"Setting up an environment for the creation of the VM cluster"*** segment)  
+>
+> 2) A simple research online shows that GCP does not support multicast. Therefore configuration needs to be done on the nodes to enable unicast.
```

Comment:
1. Why is every line blockquoted?
1. The indentation needed for the nesting of ordered lists and unordered lists looks incorrect. Please correct it.
1. It's not necessary to number each line manually. 1. 1. 1. 1. will do.

Please read [this cheat sheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) for better ways to organise the documentation.

@youngee91 can you guide Alex?